snakemake_executor_plugin_slurm/efficiency_report.py

```
@@ -3,24 +3,60 @@
+            total_seconds = (
+                time_obj.hour * 3600
+                + time_obj.minute * 60
+                + time_obj.second
+                + time_obj.microsecond / 1000000
+            )
+
+            # Add days if present (datetime treats day 1 as the first day)
+            if fmt.startswith("%d-"):
+                total_seconds += time_obj.day * 86400
+
+            return total_seconds
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)